tdms/include/field.h

```diff
@@ -331,6 +341,20 @@ class Field : public Grid{
    */
   void set_values_from(Field &other);
 
+  /**
+   * @brief Computes the maximum pointwise absolute difference of the other field to this one, divided by the largest absolute value of this field's components.
+   *
+   * Specifically, we compute and return the quantity
+   * $$ \frac{ \max_{i,j,k}(this[k][j][i] - other[k][j][i]) }{ \max_{i,j,k}this[k][j][i] } $$
```

Comment:
```suggestion
   * \f$ \frac{ \max_{i,j,k}(this[k][j][i] - other[k][j][i]) }{ \max_{i,j,k}this[k][j][i] } \f$
```
Does that render in doxygen? I _think_ you need `\f`s.